lib/PINNBench/Burgers/requirements.txt

```diff
@@ -1,226 +1,7 @@
-ruamel.yaml==0.17.21
-ruamel.yaml.clib==0.2.7
-SALib==1.4.5
-scikit-learn @ file:///Users/runner/miniforge3/conda-bld/scikit-learn_1619628646727/work
-scikit-optimize==0.9.0
-scipy==1.9.2
-scs==3.2.0
-seaborn @ file:///home/conda/feedstock_root/build_artifacts/seaborn-split_1629095986539/work
-semver==2.13.0
-Send2Trash @ file:///home/conda/feedstock_root/build_artifacts/send2trash_1624366715919/work
-six @ file:///home/conda/feedstock_root/build_artifacts/six_1620240208055/work
-sklearn==0.0
-snorkel==0.7.0b0
-SQLAlchemy @ file:///Users/runner/miniforge3/conda-bld/sqlalchemy_1624920677892/work
-statsmodels @ file:///Users/runner/miniforge3/conda-bld/statsmodels_1612273629798/work
-tables @ file:///Users/runner/miniforge3/conda-bld/pytables_1610156098887/work
-tensorboard==2.10.1
-tensorboard-data-server==0.6.1
-tensorboard-plugin-wit @ file:///home/conda/feedstock_root/build_artifacts/tensorboard-plugin-wit_1611075653546/work/tensorboard_plugin_wit-1.8.0-py3-none-any.whl
-tensorflow-addons @ https://github.com/apple/tensorflow_macos/releases/download/v0.1alpha2/tensorflow_addons_macos-0.1a2-cp38-cp38-macosx_11_0_arm64.whl
-tensorflow-estimator==2.10.0
-tensorflow-macos==2.10.0
-tensorflow-metal==0.6.0
-tensorflow-probability==0.18.0
-termcolor==1.1.0
-terminado @ file:///Users/runner/miniforge3/conda-bld/terminado_1623365427724/work
-testpath @ file:///home/conda/feedstock_root/build_artifacts/testpath_1621261527237/work
-texttable==1.6.4
-threadpoolctl @ file:///tmp/tmp79xdzxkt/threadpoolctl-2.1.0-py3-none-any.whl
-tinydb==4.7.0
-toml==0.10.2
-tomli==2.0.1
-torch==1.9.0
-torch-geometric==2.0.3
-torch-optimizer==0.3.0
-torch-scatter==2.0.9
-torch-sparse==0.6.12
-torchaudio==0.9.0
-torchvision==0.10.0
-tornado @ file:///Users/runner/miniforge3/conda-bld/tornado_1625488911690/work
-tqdm==4.64.1
-traitlets @ file:///home/conda/feedstock_root/build_artifacts/traitlets_1602771532708/work
-typed-ast==1.5.4
-typeguard @ file:///home/conda/feedstock_root/build_artifacts/typeguard_1622818924333/work
-typer==0.6.1
-typing-extensions @ file:///home/conda/feedstock_root/build_artifacts/typing_extensions_1622748266870/work
-urllib3 @ file:///home/conda/feedstock_root/build_artifacts/urllib3_1622056799390/work
-virtualenv==20.16.6
-vtk==9.0.1
-wcwidth @ file:///home/conda/feedstock_root/build_artifacts/wcwidth_1600965781394/work
-webencodings==0.5.1
-Werkzeug @ file:///home/conda/feedstock_root/build_artifacts/werkzeug_1621518206714/work
-wrapt @ file:///Users/runner/miniforge3/conda-bld/wrapt_1610094918983/work
-xlrd @ file:///home/conda/feedstock_root/build_artifacts/xlrd_1610224409810/work
-xmltodict==0.13.0
-yacs==0.1.8
-yarl @ file:///Users/runner/miniforge3/conda-bld/yarl_1610354163380/work
-yellowbrick @ file:///home/conda/feedstock_root/build_artifacts/yellowbrick_1620328308674/work
-zipp @ file:///home/conda/feedstock_root/build_artifacts/zipp_1614945704755/work
+phi==0.6.7
+phiflow==2.2.5
+tensorflow==2.10.1
+tensorflow_macos==2.10.0
```

Comment:
so this benchmark is unusable on something else than macOS?

Reply:
I removed the tf_macos requirement. It should work on other non-mac machines.